src/modules/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback, useContext, useRef } from 'react';
+import { block } from 'bem-cn';
+
+import { Button, Icon } from '@/components/ui';
+import PageContext from '@/layout/Page/PageContext';
+import Nav from '@/modules/Nav';
+import useFocusTrapping from '@/shared/utils/useFocusTrapping';
+import useKeydown from '@/shared/utils/useKeydown';
+
+import menuSvg from 'bootstrap-icons/icons/list.svg';
+import closeSvg from 'bootstrap-icons/icons/x-lg.svg';
+
+const b = block('sidebar');
+const SIDEBAR_SHOW_EVENT = 'sidebarshow';
+const SIDEBAR_HIDE_EVENT = 'sidebarhide';
```

Comment:
❗️
Давай константы наверх перенесём
```suggestion
const SIDEBAR_SHOW_EVENT = 'sidebarshow';
const SIDEBAR_HIDE_EVENT = 'sidebarhide';

const b = block('sidebar');
```